### PR TITLE
SPCTR-2320 Fix: Heading - disabling heading and sub-heading hides the separator.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-2320

### Screenshots

- https://share.getcloudapp.com/bLuOvJLE

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Added to the card.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/advanced-heading/deprecated.js [hunk 1]
```diff
@@ -414,8 +415,12 @@ const deprecated = [
 				{ descText }
 			</div>
 		);
-	}
-}
+	},
+	},
```
Comment (bsf-zanev) [inline src/blocks/advanced-heading/deprecated.js:419]: Why are the closing brackets on 418 and 419 in the same charater position?

**Review by bsf-zanev — CHANGES_REQUESTED** [review]
Please address the comments below

File: src/blocks/advanced-heading/deprecated.js (hunk 1 above)
Comment (AbhijitNage123) [reply src/blocks/advanced-heading/deprecated.js:419]: one for save() and one for to close specific deprecated code. Please check - https://share.getcloudapp.com/6quN1nzp

Comment (bsf-zanev) [reply src/blocks/advanced-heading/deprecated.js:419]: Thanks for the clarification!

As shown in the screenshot, the bracket at 218 should be 1 tab space ahead to match with the one at line 365.

Comment (AbhijitNage123) [reply src/blocks/advanced-heading/deprecated.js:419]: Resolved.

**Review by bsf-zanev — APPROVED** [review]
Looks good to me.

**Review by supriyajjadhav — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**